examples/gRPC/config.cmake

```diff
@@ -0,0 +1,2 @@
+hunter_config(ZLIB VERSION 1.2.8-p3
+              CMAKE_ARGS CMAKE_POSITION_INDEPENDENT_CODE=TRUE)
```

Comment:
Move to toolchain

Reply:
Are you suggesting I do this in my fork of gRPC or in the `cmake/project/gRPC` directory of Hunter?

Reply:
Or actually, I guess the third option is `examples/gRPC/config.cmake`.

Reply:
> Are you suggesting I do this in my fork of gRPC or in the cmake/project/gRPC directory of Hunter?

Neither of this will work.

Reply:
I put it in the example folder, and now I get this when building OpenSSL:

`target already defined - darwin64-x86_64-cc (offending arg:  -fPIC)`

Reply:
I'm guessing this relates to the custom schemes involved in building OpenSSL?

Reply:
> relates to the custom schemes involved in building OpenSSL?

Yes, I guess OpenSSL build scheme can't handle `-fPIC` compiler flag. If you don't know how to fix this one let's go back to `CMAKE_ARGS`.